.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+default_language_version:
+    python: python3    
+repos:
+  - repo: https://github.com/psf/black
```

Comment:
Any potential conflict running all three tools? Looks like the author of black has atleast attempted to address black/isort interoperability (https://github.com/timothycrosley/isort/issues/694), but the issue still remains open. It seems flake8 has plugins for both black (https://pypi.org/project/flake8-black/) and isort (https://pypi.org/project/flake8-isort/), could consider using those instead? In any case, if the current setup works this PR LGTM and we can later address problems if they happen to occur.

Reply:
In the `pre-commit` config I wanted to do somewhat the same what is being done in Travis which is to run the tools separately, just so that you can catch any issues before Travis will eventually catch them. Travis configuration could of course also be refactored, but currently it's the same that is used in other backend projects as well. `setup.cfg` also contains some configuration for `flake8` and `isort` which should make them behave in an acceptable manner for `black`.

Reply:
Sounds good to me.